src/Tribe/Editor.php

```diff
@@ -60,49 +67,26 @@ public function is_wp_version() {
 	}
 
 	/**
-	 * Checks if we have Gutenberg Project online, only useful while
-	 * its a external plugin
+	 * Checks if we have the Gutenberg Project plugin active.
 	 *
 	 * @since 4.8
 	 *
 	 * @return boolean
 	 */
-	public function is_gutenberg_active() {
-		return function_exists( 'the_gutenberg_project' );
+	private function is_gutenberg_active() {
```

Comment:
We can't change the visibility of this function without risking breakages for folks.
```suggestion
	public function is_gutenberg_active() {
```